consensus/src/epoch_manager.rs

```diff
@@ -1512,6 +1513,46 @@ impl<P: OnChainConfigProvider> EpochManager<P> {
         Ok(())
     }
 
+    fn check_author(&mut self, peer_id: AccountAddress, msg: &ConsensusMsg) -> anyhow::Result<()> {
```

Comment:
Since this a bug fix, how about a test that verifies the bug is actually fixed ?